libs/sdk-backend-tiger/src/backend/organization/styling.ts

```diff
@@ -18,31 +19,48 @@ export class OrganizationStylingService implements IOrganizationStylingService {
         );
     }
 
+    public async getActiveTheme(): Promise<ObjRef | undefined> {
+        return await this.authCall((client) =>
+            client.entities
+                .getAllEntitiesOrganizationSettings({ filter: "id==activeTheme" })
+                .then((result) => {
+                    const { data } = result;
+
+                    if (data.data.length) {
+                        return idRef((data.data[0].attributes?.content as JsonApiId).id);
+                    }
+
+                    return undefined;
+                }),
+        );
+    }
+
     public async setActiveTheme(themeRef: ObjRef): Promise<void> {
         const themeId = await objRefToIdentifier(themeRef, this.authCall);
 
         // It is not possible to PUT activeTheme if it does not exist already,
         // therefore we first try to DELETE it and POST a new one
-        try {
-            await this.authCall((client) =>
-                client.entities.deleteEntityOrganizationSettings({
-                    id: "activeTheme",
-                }),
-            );
-        } finally {
-            await this.authCall((client) =>
-                client.entities.createEntityOrganizationSettings({
-                    jsonApiOrganizationSettingInDocument: {
-                        data: {
-                            type: "organizationSetting",
-                            id: "activeTheme",
-                            attributes: {
-                                content: { activeTheme: themeId },
-                            },
+        await this.deleteActiveTheme();
+        await this.authCall((client) =>
+            client.entities.createEntityOrganizationSettings({
+                jsonApiOrganizationSettingInDocument: {
+                    data: {
+                        type: "organizationSetting",
+                        id: "activeTheme",
+                        attributes: {
+                            content: { id: themeId, type: "theme" },
                         },
                     },
-                }),
-            );
-        }
+                },
+            }),
+        );
+    }
+
+    public async deleteActiveTheme(): Promise<void> {
```

Comment:
This name is probably misleading - it does not delete the theme at all, it just resets some organization setting.
Should it be rather `resetActiveTheme`, `clearActiveTheme`,  `unsetActiveTheme`  or something similar?